components/accordion/stories/accordion.stories.js

```diff
@@ -1,5 +1,6 @@
 // Import the component markup template
 import { Template } from "./template";
+import { withActions } from '@storybook/addon-actions/decorator';
```

Comment:
Global decorators should go in the preview.js file for Storybook - see line 215 in preview.js:

```diff
export const decorators = [
  withTextDirectionWrapper,
  withLanguageWrapper,
  withReducedMotionWrapper,
  withContextWrapper,
+ withActions,
];
```